zip-archive.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                zip-archive
-Version:             0.4.1
+Version:             1.0.0
```

Comment:
Version would bump to 0.5.0.  Remember that in the Haskell versioning policy, it's EPOCH.MAJOR.MINOR.PATCH.